app/assets/javascripts/services/caseCSVSvc.js

```diff
@@ -351,35 +377,41 @@
          */
 
         function stringifySnapshot (aCase, snapshot, withHeader) {
-          var csvContent  = '';
-
+          let infoArray = [];
+          let matchingQuery = null;
+          let matchingQueryText = null;
+          const snapshotName = snapshot.name();
+          const snapshotTime = snapshot.time;
+          const caseNumber = aCase.caseNo;
+          let csvContent = '';
+          let queryIdAsInt = 0;
```

Comment:
do we actually have to provide a default like this?   Versus defining it where we use it?